### ISSUE TITLE
Server title and hostname empty on create

### ISSUE THREAD

**Issue by scop** [UpCloudLtd/upcloud-whmcs-module#46]
When a server (vm) create is attempted using the UpCloud API, the title and hostname are sent over as empty, causing the creation to fail.

Module logs shows request payload starting with
```
server
{
    "server": {
        "metadata": "yes",
        "zone": "fi-hel2",
        "title": "",
        "hostname": "",
```
Both title and hostname must be populated.

This quick hack works around the issue, but I'm not sure if it's the appropriate fix.
```diff
--- lib/vmManager.php
+++ lib/vmManager.php
@@ -64,3 +64,3 @@
     $user_data = $this->params['customfields']['userData'];
-    $Hostname = $this->params['domain'] ?? 'client' . $this->params['serviceid'] . '.' . $_SERVER['SERVER_NAME'];
+    $Hostname = 'client' . $this->params['serviceid'] . '.' . $_SERVER['SERVER_NAME'];
     $sshkey = empty($sshkey) ? "na" : $sshkey;
```
I.e. `$this->params['domain'] ?? ` was removed.

**Comment by adityawbwce** [comment]
Change the line number #65 value from
$Hostname = $this->params['domain'] ?? 'client' . $this->params['serviceid'] . '.' . $_SERVER['SERVER_NAME'];
to
$Hostname = !empty($this->params['domain']) ? $this->params['domain'] : 'client' . $this->params['serviceid'] . '.' . $_SERVER['SERVER_NAME'];

**Comment by scop** [comment]
Fixed by #60
